From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @eslesar, has been notified to review your proposed change.

MGoedtel commented:
@4c74356b41 thank you for taking the time to submit a request to make corrections/updates to this article.  I have reviewed them and approve them to be merged.  @sign-off

v-viinde committed:
USER STORY 1228776 SaaS App Tutorial: Amazon Web Services (AWS) Update

v-viinde committed:
Merge branch 'master' into awsupdate

msmbaldwin committed:
Updated author fields

v-viinde committed:
USER STORY 1231248 SaaS App Tutorial: Dynamic Signal

v-nagta committed:
USER STORY 1232759 SaaS App Tutorial: Workday Tutorial Update

v-viinde committed:
Updated

roygara committed:
Updating image and text.

twooley committed:
[Azure AD B2B] New article about allow and deny lists

twooley committed:
[Azure AD B2B] New article about allow and deny lists

twooley committed:
[Azure AD B2B] New article about allow and deny lists

twooley committed:
[Azure AD B2B] New article about allow and deny lists

twooley committed:
[Azure AD B2B] New article about allow and deny lists

MGoedtel committed:
updated article with better example

MGoedtel committed:
added warning and updated comment in json

MGoedtel committed:
minor edit

MGoedtel committed:
corrected link issue and list items

MGoedtel committed:
update formatting of code

PRMerger17 committed:
Merge pull request #37436 from msmbaldwin/mdcleanup

Updated author fields

MGoedtel committed:
updated codeblock

JasonWHowell committed:
fixing formula backgrounds

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#6612

slack committed:
doc(aks): add preview banner to FAQ

PRMerger12 committed:
Merge pull request #37671 from tfitzmac/0412admin

fixes MicrosoftDocs/azure-docs#6612

Commit:
Mior updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aas0412

daveba committed:
update TOC

PRMerger17 committed:
Merge pull request #37674 from Minewiskan/aas0412

Aas0412

PRMerger20 committed:
Merge pull request #37678 from slack/aks-preview-banner

doc(aks): add preview banner to FAQ

itechedit committed:
Merge pull request #37656 from MGoedtel/UpdateLATemplate411

Update la template411

itechedit committed:
Merge pull request #37672 from JasonWHowell/formulas

fixing formula backgrounds

johnkemnetz committed:
Update monitoring-diagnostic-logs-schema.md

v-shils committed:
Merge pull request #37676 from daveba/msi-toc-update

[Azure AD MDI] Update TOC

Commit:
Updating operator TOC for redirected article

PRMerger14 committed:
Merge pull request #37683 from johnkemnetz/patch-2

Update monitoring-diagnostic-logs-schema.md

twooley committed:
Added TOC entry

MicrosoftGuyJFlo committed:
[Azure AD] SSPR - Issue 6994

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-metadata-updates

msjuergent committed:
add_us_east_2

v-shils committed:
Merge pull request #37685 from jeffgilb/412

Updating operator TOC for redirected article

nitinme committed:
Fixing HDI R metadata

Commit:
Merge pull request #37689 from MicrosoftGuyJFlo/Issue6994

[Azure AD] SSPR - Issue 6994

nitinme committed:
fixing another tag

nitinme committed:
fixing interactive query metadata tags

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-metadata-updates

rjagiewich committed:
Merge pull request #37623 from twooley/twb2ballowlist

[Azure AD B2B] New article about allow and deny lists

Commit:
Merge pull request #37690 from msjuergent/wadoc

add_us_east_2

carolinacmoravia committed:
CC53147:  Extra bracket in structure

Hello, @tdykstra,

This proposed file change comes from this Community Contribution:  https://github.com/MicrosoftDocs/azure-docs.ja-jp/pull/477/files

Please, help to check this proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

PRMerger12 committed:
Merge pull request #37692 from nitinme/nitinme-metadata-updates

Fixing HDI R metadata

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-metadata-updates

nitinme committed:
Hadoop metadata update - batch 1

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-metadata-updates

PRMerger17 committed:
Merge pull request #37696 from nitinme/nitinme-metadata-updates

Hadoop metadata update - Batch 1

nitinme committed:
Hadoop metadata updates

PRMerger18 committed:
Merge pull request #37694 from carolinacmoravia/patch-679

CC53147:  Extra bracket in structure

cherylmc committed:
update

cherylmc committed:
update

Commit:
Merge pull request #37700 from nitinme/nitinme-metadata-updates

Hadoop metadata updates - batch 2

cherylmc committed:
update title

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Ja-Dunn committed:
Merge pull request #37701 from cherylmc/toc

update

PRMerger16 committed:
Merge pull request #37227 from v-viinde/awsupdate

USER STORY 1228776 SaaS App Tutorial: Amazon Web Services (AWS) Update

rjagiewich committed:
Merge pull request #37606 from roygara/fixImage

Updating image and text.

v-alje committed:
Merge pull request #37710 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #37547 from v-nagta/workdayupdate

USER STORY 1232759 SaaS App Tutorial: Workday Tutorial Update

rjagiewich committed:
Merge pull request #37460 from v-viinde/dynamicsignal

USER STORY 1231248 SaaS App Tutorial: Dynamic Signal

PRMerger20 committed:
Merge pull request #37708 from cherylmc/patch-2

update title
